build/transactions/gas-calculation.mdx

```diff
@@ -0,0 +1,36 @@
+# Qi Transactions
+
+## Gas Calculation
+
+Gas calculation for Qi Transactions is unique and takes into account several factors specific to Quai Network's architecture. Here's an overview of how gas is calculated:
+
+### Total Gas Calculation
+
+The total gas for a Qi transaction is calculated by considering several components:
+
+1. **Intrinsic Gas**: This is a base cost associated with the transaction itself.
+2. **Input and Output Costs**: Each input and output in the transaction incurs a gas cost.
+3. **Cross-Shard Operations**: Additional gas is charged for operations that cross shard boundaries.
+4. **Conversions**: If the transaction involves converting between different types of assets, this incurs extra gas costs.
+
+### Intrinsic Gas
+
+The intrinsic gas calculation takes into account:
+- The number of inputs in the transaction
+- The number of outputs in the transaction
```

Comment:
The gas required is technically based on the size of the inputs and outputs in bytes rather than just the number of inputs and outputs. The statement does however convey the correct info since the size of each input and output is more of less fixed. Either way of phrasing is honestly okay with me.